apps/desktop/src/features/board/board.css

```
@@ -0,0 +1,43 @@
+.board-hover-menu-morph {
+  --board-menu-padding: var(--space-1);
+  --board-menu-icon-row-size: 40px;
+  --board-menu-icon-gap: 0px;
+  --board-menu-content-gap: var(--space-2);
+  --board-menu-content-inline-padding: var(--space-0);
+  --board-menu-collapsed-height: calc(var(--board-menu-icon-row-size) + (var(--board-menu-padding) * 2) + 2px);
+  --board-menu-collapsed-width: calc(
+    (var(--board-menu-icon-row-size) * 1) + var(--board-menu-icon-gap) + (var(--board-menu-padding) * 2) +
```

Stylelint (17.11.1)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)